artifact_list_builder.py

```diff
@@ -671,6 +675,72 @@ def _listRemoteRepository(self, repoUrl, classifiersFilter, prefix=""):
             self._addArtifact(artifacts, gav[0], gav[1], gav[2], gavExtClass[gav], suffixes.get(gav), repoUrl)
         return artifacts
 
+    def _listIndyRepository(self, repoUrl, classifiersFilter, prefix=""):
+        logging.debug("Listing Indy remote repository %s prefix '%s'", repoUrl, prefix)
+        url = repoUrl.replace('indy://', 'http://').replace('indys://', 'https://') + prefix
+
+        try:
+            response = requests.get(url)
+            if response.status_code != 200:
+                logging.warning("Cannot retrieve listing for: " + url + ". Error: " + str(response))
+                return {}
+            else:
+                out = response.json()
+                # logging.debug(out)
+        except IOError as err:
+            if prefix:
+                logging.warning(str(err))
+                out = {}
+            else:
+                raise err
+
+        # ^/(groupId)/(artifactId)/(version)/(filename)$
+        regexGAVF = re.compile(r'/(.+)/([^/]+)/([^/]+)/([^/]+\.[^/.]+)$')
+        gavExtClass = {}  # { (g,a,v): {ext: set([class])} }
+        suffixes = {}     # { (g,a,v): suffix }
+
+        listings = out.get('listingUrls')
+        # logging.debug(listings)
+        if listings is None or len(listings) < 1:
+            logging.warning("No results from: " + url)
+            return {}
+
+        for listing in listings:
+            line = listing['path']
+            # logging.debug("Process listing: " + line)
+            if (line):
+                gavf = regexGAVF.match(line)
+                # if gavf is None:
+                #     logging.debug("No GAVTC match for: " + line)
+                # else:
+                if gavf is not None:
+                    # logging.debug("Group 1 of regex: " + gavf.group(1))
+                    groupId = gavf.group(1).replace('/', '.')
+                    artifactId = gavf.group(2)
+                    version = gavf.group(3)
+                    filename = gavf.group(4)
+
+                    # logging.debug("Found G: %s, A: %s, V: %s, F: %s" % (groupId, artifactId, version, filename))
```

Comment:
Why are these log messages commented out? I again think that having it on debug level, which is not default, is ok.